docs/api-docs/catalog/price-list-overview.md

```diff
@@ -15,9 +15,9 @@
 ## What is a price list?
 
 
-A Price List allows you to populate different versions of catalog pricing and assign them to different [Customer Groups](/api-reference/customer-subscribers/customers-api). The prices are specified exclusively at the variant level.
+A Price List allows you to populate different versions of catalog pricing and assign them to different [Customer Groups](/api-reference/customer-subscribers/customers-api). The prices are specified exclusively at the variant level. If an active Price List does not contain prices for a variant, then the Catalog pricing will be used. 
 
-If an active Price List does not contain prices for a variant, then the Catalog pricing will be used. You can associate a Price List to a Customer Group via the Control Panel or the [Customer Groups API.](/api-reference/customer-subscribers/customers-api)
+You can assign a Price List to a specific sales channel using price list assignments via the [Price List Assignment API](https://developer.bigcommerce.com/api-reference/store-management/price-lists/price-lists-assignments/createpricelistassignments). You can also associate a Price List to a Customer Group via the Control Panel or the [Customer Groups API](/api-reference/customer-subscribers/customers-api). Price list assignments combined with a customer group assignment allow you to better target specifically the logged-in customers shopping on that channel.

```

Comment:
It's not only for specific sales channels- you can assign default prices for a sales channel, or for a customer group, or for a customer group on a specific sales channel.  (I'm probably not putting that well.)

Reply:
I have updated the scope of Price List Assignments